app/extensions/safe/network/authenticator-comms.js

```diff
@@ -14,6 +14,11 @@ let peruseAppObj;
 let store;
 let browserAuthReqUri;
 
+//////////////////////////
+//
+// QUESTION: Many of these functions are duplicated in the\
+// index of this directory. What is the purpose of these?
```

Comment:
Aha, yeh. This is a hangover from me trying to find the 'correct' place for all of this. Was a bit awkward with some functionality in the main process, some in bg process etc.

So, the question probably should be... what's communicating here (the peruse app or webFetch? Are they going to be the same thing?).

Proooobably we should port the peruseApp functionality into the extension also (as that's SAFE specific, not browser generic.) So this file would be used by that app (be it registered or unregistered). 

Something like that... @hunterlester I'm open to suggestion! Now you've had a poke around and can see what's going on let's have a chat about this and what might be the best way to organise it.